tests/test_you_should_use.zunit

```diff
@@ -6,6 +6,9 @@
 }
 
 @test 'ysu version exported' {
+    (( $+commands[git] )) || skip "git not found"
```

Comment:
I elected to use pure Zsh when checking if a command is present to avoid a dependency on `which`.